etc/config-template.toml

```diff
@@ -111,6 +111,13 @@ max-manifest-file-size = "20MB"
 # If true, the database will be created if it is missing.
 create-if-missing = true
 
+# This specifies the info LOG dir.
+# If it is empty, the log files will be in the same dir as data.
+# If it is non empty, the log files will be in the specified dir,
+# and the db data dir's absolute path will be used as the log file
```

Comment:
Let's say `log-dir = "/path/to/dir"`, then the actual LOG file is named `path_to_dir_LOG`, and it is stored under `/path/to/dir`.